tests/Fixtures/TestBundle/DataProvider/TaxonItemDataProvider.php

```diff
@@ -41,6 +44,8 @@ public function supports(string $resourceClass, string $operationName = null, ar
      */
     public function getItem(string $resourceClass, $id, string $operationName = null, array $context = [])
     {
-        return $this->managerRegistry->getRepository(Taxon::class)->find($id);
+        return $this->managerRegistry->getRepository($this->orm ? Taxon::class : TaxonDocument::class)->findOneBy([
```

Comment:
Maybe we can have a `get_class` for the manager registry instead of `$orm`?

Reply:
The problem is, we cannot tell whether we're using the ORM or ODM, as it's the same interface as `$resourceClass` for both.

Reply:
Oh, I get what you mean now. But that seems hacky.

Reply:
And in fact, both are https://github.com/symfony/symfony/blob/v4.3.2/src/Symfony/Bridge/Doctrine/ManagerRegistry.php so it doesn't help.